external/WICED/bcm4390x/include/wwd_constants.h

```diff
@@ -0,0 +1,175 @@
+/****************************************************************************
+ *
+ * Copyright 2017, Cypress Semiconductor Corporation or a subsidiary of
+ * Cypress Semiconductor Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/** @file
+ *  Defines common constants used with WICED
+ *
+ */
+
+#ifndef INCLUDED_WWD_CONSTANTS_H_
+#define INCLUDED_WWD_CONSTANTS_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef RESULT_ENUM
+#define RESULT_ENUM( prefix, name, value )  prefix ## name = (value)
+#endif							/* ifndef RESULT_ENUM */
+
+/* These Enum result values are for WWD errors
+ * Values: 1000 - 1999
+ */
+
+#define WWD_RESULT_LIST( prefix )  \
+    RESULT_ENUM( prefix, SUCCESS,                         0 ),   /**< Success */                           \
```

Comment:
Spaces may cause violation of coding rule. Let's use a tab at start of line.

Reply:
it will be changed as your guidance.

BR, HJ Lee.